3-mcstas/SANS_from_function.ipynb

```diff
@@ -519,22 +590,20 @@
    "execution_count": null,
    "id": "lesser-retrieval",
    "metadata": {
-    "editable": true,
     "slideshow": {
      "slide_type": ""
     },
     "tags": []
    },
    "outputs": [],
    "source": [
-    "# quiz(instrument)"
+    "quiz.question_7(instrument)"
```

Comment:
This raised an error:
![Screenshot at 2023-09-03 08-21-13](https://github.com/ess-dmsc-dram/dmsc-school/assets/39047984/a886ad77-21cd-4ef2-8c10-3cc6cc8ee5d1)

Incidently, the rest of the notebook did not run.

Reply:
Fixed this on the fixes-for-mads branch.